### PR TITLE
gives darkspawn a hint

### PR DESCRIPTION
**Pull request by Sniblet**
PSST!! PRESS THIS BUTTON TO WIN!!

# Document the changes in your pull request

Slightly extends the pass ability's tooltip

# Changelog
:cl:  
tweak: slightly expanded the tooltip for pass
/:cl:

### CODE REVIEW

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
what's "them"? reading this, I do not know what to examine, just that I have to examine something. Do I examine the tendrils? My active arm? Shift click the action in the top left?

**Comment by Sniblet** [comment]
Tendrils is the direct object of the sentence, if someone can’t infer that it’s referring to the tendrils I don’t think I can help them (in this sentence “them” refers to “someone”)

**Review by adamsong — APPROVED** [review]
Should be fine

**Review by JamieD1 — APPROVED** [review]

File: yogstation/code/modules/antagonists/darkspawn/darkspawn_abilities/pass.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /datum/action/innate/darkspawn/pass
 	name = "Pass"
 	id = "pass"
-	desc = "Twists an active arm into tendrils with many uses."
+	desc = "Twists an active arm into tendrils with many important uses, examine them to see a list."
```
Comment (Bibby0110) [inline yogstation/code/modules/antagonists/darkspawn/darkspawn_abilities/pass.dm]:
```suggestion
	desc = "Twists an active arm into tendrils with many important uses. Examine the tendrils to see a list of uses."
```

**Review by Bibby0110 — DISMISSED** [review]
I have to agree with baimou. This can be confusing

**Review by SomeguyManperson — APPROVED** [review]
Honestly should have added this at the start since its slightly more lilely to be rewd here than in the chat

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**